pkg/capabilities/consensus/ocr3/data_feeds_consensus_builders_generated.go

```diff
@@ -9,8 +9,7 @@ import (
 )
 
 
-
-func (cfg ConsensusConfig) New(w *workflows.WorkflowSpecFactory,ref string, input ConsensusInput)SignedReportCap {
+func (cfg DataFeedsConsensusConfig) New(w *workflows.WorkflowSpecFactory,ref string, input DataFeedsConsensusInput)SignedReportCap {
```

Comment:
This file doesn't open automatically, but the magic lives here

ocr provides many consensus algorithms.  This allows 

ocr3.DataFeedsConsensusConfig{...}.New() 

along with other types like

ocr3.IdenticalConsensusConfig{...}.New()

The common allows us to share SignedReportCap since they both return the same type.  Allowing a .common makes it easy to share without one of the implementations owning the type.